homeassistant/components/rfxtrx/config_flow.py

```diff
@@ -159,13 +166,17 @@ async def async_step_prompt_options(self, user_input=None):
             step_id="prompt_options", data_schema=vol.Schema(options), errors=errors
         )
 
-    async def async_step_set_device_options(self, user_input=None):
+    async def async_step_set_device_options(
+        self, user_input: dict[str, Any] | None = None
+    ) -> data_entry_flow.FlowResult:
         """Manage device options."""
         errors = {}
+        devices: dict[str, dict[str, Any] | None] = {}
```

Comment:
These variables seem to be used only within the scope of `if user_input is not None`. Can the variable be declared inside to make its scope as small as possible?